src/api/app/views/webui2/webui/user/_breadcrumb_items.html.haml

```diff
@@ -1,9 +1,18 @@
-- if current_page?(users_path)
+- case action_name
+- when 'index'
   -# This page is only reachable from the configuration, so it's based on configuration
   = render partial: 'webui/configuration/breadcrumb_items'
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
     Manage Users
-- else
-  = render partial: 'webui/main/breadcrumb_items'
+- when 'edit'
+  = render partial: 'webui/configuration/breadcrumb_items'
+  %li.breadcrumb-item
+    = link_to('Manage users', users_path)
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-    Home of #{@displayed_user}
+    = @displayed_user
```

Comment:
For the groups, I'm doing this when we are on the `Edit Group` page.

`Home (link) > Manage Groups (link) > Group Name (link) > Edit`.

I think we should align here. Which way do you prefer?

Reply:
That's shown on the user's home page, like [here](https://build.opensuse.org/user/show/bgeuken). On the configuration page it would be `Configuration > Manage Users` and `Configuration > Manage Users > tux` (I changed it to edit instead of tux, aka the user name).
I think the configuration part is fine, since this is part of the admin configuration. Users would use a different view (dialog actually) to configure their account.

Reply:
Oh my bad, thank you for the explanation.